tools/cli/src/sourcemaps/add-sources.ts

```diff
@@ -0,0 +1,118 @@
+import { AsyncResult, DebugIdGenerator, Err, SourceProcessor, flatMap } from '@backtrace/sourcemap-tools';
+import { GlobalOptions } from '..';
+import { Command } from '../commands/Command';
+import { ContentFile, failIfEmpty, map, parseJSON, pass, passOk, readFile, writeFile } from '../helpers/common';
+import { find } from '../helpers/find';
+import { CliLogger, createLogger } from '../logger';
+
+interface AddSourcesOptions extends GlobalOptions {
+    readonly path: string[];
+    readonly 'dry-run': boolean;
+    readonly force: boolean;
+    readonly skipFailing: boolean;
+    readonly 'pass-with-no-files': boolean;
+}
+
+type ObjectFile = readonly [object, string];
+
+export const addSourcesCmd = new Command<AddSourcesOptions>({
+    name: 'add-sources',
+    description: 'Adds sources to sourcemap files',
+})
+    .option({
+        name: 'path',
+        description: 'Path to sourcemap files to append sources to.',
+        defaultOption: true,
+        defaultValue: process.cwd(),
+        multiple: true,
+        alias: 'p',
+    })
+    .option({
+        name: 'dry-run',
+        alias: 'n',
```

Comment:
Is 'n' an existing convention? What about 'd'?

Reply:
I believe so. Most places I've seen have `-n` alias as the dry run switch.

E.g. `rsync`:
```
-n, --dry-run               show what would have been transferred
```